techmatters/flex-plugins
# CHI-959: Fix import default resources script for modiules

## Description

* Moved createTwilioApiKeyAndSsmSecret to be a twilioResources command
* Fixed import-default-account to work with module structure

### Checklist
- [ X ] Corresponding issue has been opened
- [ N/A ] New tests added
- [ N/A ] Strings are localized


### Related Issues
Fixes CHI-959

### Verification steps

* Run scripts as part of a setup according to the README
* verify that the API Keys & SSM params are present as expected
* Verify `terraform plan` is creating the expected resources after the import defaults has been run
